runtime/oti/j9vm.hdf

```diff
@@ -1341,6 +1341,8 @@ typedef UDATA (* lookupNativeAddressCallback)(struct J9VMThread *currentThread,
 		</description>
 		<struct>J9VirtualThreadStartedEvent</struct>
 		<data type="struct J9VMThread*" name="currentThread" description="current thread" />
+		<data type="jobject" name="thread" description="the virtual thread" />
```

Comment:
we will need to pass the thread object to the `J9HOOK_VM_THREAD_*` hooks so that they can use the new TLS for JDK19+.

Reply:
We will not be using `J9HOOK_VM_THREAD_CREATED` since the data is lazy allocated. But for `J9HOOK_VM_THREAD_DESTROY` do we really need to pass the thread object? Can't we fetch it from `currentThread->carrierThreadObject`

Reply:
`carrierThreadObject` should work for platform threads.